src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -9584,6 +9588,12 @@ namespace AirflowNetworkBalanceManager {
             // Skip the inlet and outlet nodes of zone dehumidifiers
             if (GetZoneDehumidifierNodeNumber(state, i)) NodeFound(i) = true;
 
+            // Skip HPWH nodes that don't have to be included in the AFN
+            if (HeatPumpWaterHeaterNodeAFNExceptions(state, i)) NodeFound(i) = true;
+
+            // Skip Standalone ERV nodes that don't have to be included in the AFN
+            if (StandAloneERVAFNExceptions(state, i)) NodeFound(i) = true;
+
```

Comment:
Add similar logic than `GetZoneDehumidifierNodeNumber` that provides an exception for reasonable standalone and HPWH configurations that can be modeled along AFN without having to be part of the AFN.

Reply:
Functionalities of GetZoneDehumidifierNodeNumber (exist), HeatPumpWaterHeaterNodeAFNExceptions (new), and StandAloneERVAFNExceptions (new) are the same. However, the function names are much different. I prefer to have similar function names, so that it will be easy for people to recognize the same functionality.